vsphere-manager-blueprint-inputs.yaml

```diff
@@ -0,0 +1,235 @@
+#############################
+# Vsphere air specific inputs
+#############################
+
+# Credentials:
+#vsphere_username:
+#vsphere_password:
+#vsphere_host:
+#vsphere_port: 443
+#vsphere_datacenter_name: Datacenter
+#vsphere_resource_pool_name: Resources
+#vsphere_auto_placement: true
+
+# Template
+#template:
+
```

Comment:
please supply as many default values as possible so that the user doesn't have to guess what to put in each input.